content/code-examples/environments/create-environment.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Novu create one environment
+id: post-/v1/environments/
+type: snippets
+---
+
+```javascript label=Node.js
+import { Novu } from '@novu/node';
+
+export const novu = new Novu('<REPLACE_WITH_API_KEY_FROM_ADMIN_PANEL>');
+
+const payload: IEnvironmentCreatePayload = {
+  name: 'test',
+  parentId: 'as20dfbjsfi3ssdk39dh'
+} 
+
+await novu.environments.create(payload);
```

Comment:
```suggestion
await novu.environments.create({
  name: 'test',
  parentId: 'as20dfbjsfi3ssdk39dh'
});
```

Reply:
I do not find that bad to separate the payload. It also adds the type of the payload that helps as a referral to check in the code the potential object.

Reply:
No problem, but we should do it in the rest of the docs then